pkg/agent/datamodel/sig_config.go

```diff
@@ -439,6 +439,10 @@ const (
 	// DO NOT MODIFY: used for freezing MarinerV2KataGen2TL.
 	FrozenCBLMarinerV2KataGen2TLSIGImageVersion = "2022.12.15"
 
+	// DO NOT MODIFY: Used for freezing Kata VHD for kata-cc runtime, until the image is deprecated.
+	FrozenCBLMarinerV2KataGen2SIGImageVersion string = "202508.06.0"
+	FrozenAzureLinuxV2KataGen2SIGImageVersion string = "202508.06.0"
```

Comment:
I think 202508.06 should be the right date since I'm picking the release from this [PR](https://dev.azure.com/msazure/CloudNativeCompute/_git/aks-rp/pullrequest/13230704?_a=files) but please correct me if a different date should be used.

Reply:
@Bickor I'd recommend using `202507.21.0` instead to ensure you're not inheriting any bugs introduced in a few of the latest node images

Reply:
do we also need to freeze kataTL?